docs/docs/concepts/default-actions.mdx

```diff
@@ -135,128 +135,3 @@ class ActionSessionStart(Action):
         # as a user message follows
         return [SessionStarted(), ActionExecuted("action_listen")]
 ```
-
-## action_default_fallback
-
-This action undoes the last user-bot interaction and sends the `utter_default` response if it is defined.
```

Comment:
@dakshvar22 we do still have a fallback, no? I thought I saw a ticket for handling this in docsearch

Reply:
Yes, we plan to have it. I also understood today from Thomas that it is not being triggered anywhere yet but it needs to in specific scenarios.

Reply:
the current docs for this currently refer to the [fallback-handhoff page](https://rasa.com/docs/rasa/fallback-handoff) which is currently written in the context of NLU-based assistants. @dakshvar22 can you make sure that when this is implemented, the docs are updated to reflect the way it works with CALM?

Reply:
yep, will take it as part of this ticket - https://rasahq.atlassian.net/browse/ENG-579